### DISCUSSION TITLE
[Bug]: Landscape is not working

### DISCUSSION THREAD

**Discussion by Lintume** [spatie/laravel-pdf#30] [General]
### What happened?

I am trying to do a pdf in landscape format, but it always returns normal.

this is the code:
` Pdf::view('documents.articles.pdf', ['article' => $this->article])
            ->landscape()
            ->disk('public')
            ->save($path);`

### How to reproduce the bug

When I generate doc using code above I'm getting portrait orientation instead landscape.

### Package Version

1.0.0

### PHP Version

8.1

### Laravel Version

10.32.1

### Which operating systems does with happen with?

macOS

**Comment by freekmurze** [comment]
Our tests indicate that this is working. https://github.com/spatie/laravel-pdf/blob/main/tests/PdfTest.php#L109-L117

Could you submit a PR with a failing test?

**Comment by asemarian** [comment]
I'm having the same issue. In my case, it seems to occur only when I use custom paper dimensions, like so:
```php
return Pdf::view('pdf.test')
            ->orientation(Orientation::Landscape)
            ->name('test.pdf')
            ->paperSize(4.13, 2.91, 'in')
            ->download();
```

**Reply by asemarian** [reply]
This is probably a Browsershot/Puppeteer limitation
